lizard_auth_client/client.py

```diff
@@ -247,7 +245,7 @@ def sso_authenticate_v2(sso_server_api_start_url, sso_key, sso_secret,
             r.raise_for_status()
         return r.json()['user']
 
-    except:
+    except:  # noqa: E722
```

Comment:
According to black, the following ought to work fine, too:

```suggestion
        except Exception:
```

Reply:
Done